test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -73,7 +73,7 @@
 
     static final String PRAVEGA_OPERATOR = "pravega-operator";
     static final String CUSTOM_RESOURCE_GROUP_PRAVEGA = "pravega.pravega.io";
-    static final String CUSTOM_RESOURCE_VERSION_PRAVEGA = "v1alpha1";
+    static final String CUSTOM_RESOURCE_VERSION_PRAVEGA = "v1beta1";
```

Comment:
This seems not related to the purpose of the PR.

Reply:
System test framework failed with `String CUSTOM_RESOURCE_VERSION_PRAVEGA = "v1alpha1";` configuration, So modify this to run System test successfully

Reply:
This changes not related to PR so revert the changes